### ISSUE TITLE
[Enhancement]: EXPLOIT- Config option to make blueprints single use like they used to be.

### ISSUE THREAD

**Issue by ProsperCraft** [Brian-Wuest/MC-Prefab#266]
### Minecraft Version

1.16.5, 1.18.2

### Details

Or...not using the durability of the blueprint to determine how many times it has left.

Block that repair equipment lead to an exploit because it can be used, then repaired, thus endless use.

**Comment by kaojinn** [comment]
> ### Minecraft Version
> 1.16.5, 1.18.2
> 
> ### Forge Version
> _No response_
> 
> ### Mod Version
> _No response_
> 
> ### Details
> Or...not using the durability of the blueprint to determine how many times it has left.
> 
> Block that repair equipment lead to an exploit because it can be used, then repaired, thus endless use.

You've gotta give us something more specific to take a look at.  Which blueprint are you experiencing this issue with?  If I remember correctly, the villager housing and the bulldozer both have charges, but the bulldozer is purposely intended to be recharged via the anvil.

**Comment by ProsperCraft** [comment]
The blueprint Modern builds for example.

**Comment by Brian-Wuest** [comment]
The modern buildings blueprint is single use only.

There are only 3 multi-use builds. These are intended for multi-use.

1. Villager Houses
2. Instant Bridge
3. Bulldozer

The modern building item hasn't been multi-use for a long time now.

If other mods allow for making any item multi-use or have the ability to repair items which we did not specify repair functionality for that's on the modpack creator to disable those things or on the server administrator to disable that functionality.

Closing issue.
